astro/src/content/docs/get-started/use-cases/auth-service.mdx

```diff
@@ -0,0 +1,120 @@
+---
+title: Auth as a Service
+description: Delegate your authentication to FusionAuth.
+section: get started
+subcategory: use cases
+---
+import LoginBefore from 'src/diagrams/quickstarts/login-before.astro';
+import LoginAfter from 'src/diagrams/quickstarts/login-after.astro';
+import AuthServiceDescription from 'src/content/docs/get-started/use-cases/_auth-service-description.mdx';
+import Breadcrumb from 'src/components/Breadcrumb.astro';
+
+## Overview
+
+<AuthServiceDescription />
+
+## Problem 
+
+You have an application that you want to add authentication features to. You also want to safely store users' personal identifiable information and credentials.
```

Comment:
I really like the statement `offload authentication and add features with configuration rather than coding`. As an application developer who's new to auth it's an aha moment as I don't have to add code to my application for authentication to work.